src/fullfile.c

```diff
@@ -248,7 +251,15 @@ int download_fullfiles(struct list *files, int *num_downloads)
 		if (file->is_mix) {
 			download_mix_file(file);
 		} else {
-			download_file(download_handle, file);
+			ret = download_file(download_handle, file);
+			if (ret == -CURLE_WRITE_ERROR) {
```

Comment:
What about other curl errors? Are they ok? Why do we need a behavior here different than pack downloads?
Is this problem reproduced with packs?

Reply:
no, this problem doesn't seem to be happening with packs.

I was only looking for CURL_WRITE_ERROR in this case because at least I know that is a terminal error, and there is no point on keep retrying when that error occurs, with other errors it may be possible to keep trying I guess

Reply:
This function is used both here and in packs so I would expect the code that handles it to be similar. If the problem is not happening in packs, try to do the same error handling